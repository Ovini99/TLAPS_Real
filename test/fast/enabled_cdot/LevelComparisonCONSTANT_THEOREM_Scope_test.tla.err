@!!BEGIN
@!!type:obligationsnumber
@!!count:0
@!!END

File "/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/../library/TLAPS.tla", line 1, character 1 to line 363, character 77:
[INFO]: All 0 obligation proved.
@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:17:1:17:3
@!!status:to be proved
@!!END

@!!BEGIN
@!!type:obligation
@!!id:2
@!!loc:17:4:17:19
@!!status:to be proved
@!!END

@!!BEGIN
@!!type:obligationsnumber
@!!count:2
@!!END

** Unexpanded symbols: ---

@!!BEGIN
@!!type:obligation
@!!id:2
@!!loc:17:4:17:19
@!!status:trivial
@!!prover:tlapm
@!!already:false
@!!END

Executable "isabelle-process" not found in this PATH:
/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/lib/tlaps/bin/:/home/ovini/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/lib/tlaps/bin
@!!BEGIN
@!!type:obligation
@!!id:1
@!!loc:17:1:17:3
@!!status:proved
@!!prover:smt
@!!meth:time-limit: 25; time-used: 0.0 (0%)
@!!already:false
@!!END

File "./LevelComparisonCONSTANT_THEOREM_Scope_test.tla", line 1, character 1 to line 19, character 80:
[INFO]: All 2 obligations proved.
