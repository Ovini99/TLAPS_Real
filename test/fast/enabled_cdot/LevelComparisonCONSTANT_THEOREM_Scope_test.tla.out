
\* TLAPM version 1.5.0
\* launched at 2023-11-27 14:14:23 with command line:
\* /home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/../tlapm -I /home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/../library --toolbox 0 0 --nofp LevelComparisonCONSTANT_THEOREM_Scope_test.tla --stretch 5

(* created new ".tlacache/TLAPS.tlaps/TLAPS.thy" *)
(* fingerprints written in ".tlacache/TLAPS.tlaps/fingerprints" *)
(* will not use fingerprints (because of option `--nofp`), but will now load fingerprints from the file `.tlacache/LevelComparisonCONSTANT_THEOREM_Scope_test.tlaps/fingerprints`, in order to overwrite with the new fingerprints, and then save the results at the end. *)
COMMAND-LINE ARGUMENT
file=.tlacache/LevelComparisonCONSTANT_THEOREM_Scope_test.tlaps/tlapm_1f74b1.smt; PATH='/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main/test/lib/tlaps/bin/:/home/ovini/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/tlapm-main:/home/ovini/Downloads/tlapm_real_altered/tlapm-main/lib/tlaps/bin'; z3 -smt2 -v:0 AUTO_CONFIG=false smt.MBQI=true "$file" >.tlacache/LevelComparisonCONSTANT_THEOREM_Scope_test.tlaps/tlapm_1f74b1.smt.out
CONTENT OF FILE SENT TO BACKEND
;; TLA+ Proof Manager 1.5.0
;; Proof obligation #1
;;   generated from file "./LevelComparisonCONSTANT_THEOREM_Scope_test.tla", line 17, characters 1-2

(set-logic UFNIA)
(declare-sort u 0)
;; Standard TLA+ operators
(declare-fun int2u (Int) u)
(declare-fun u2int (u) Int)
(declare-fun tla__Int () u)
(declare-fun tla__le (u u) Bool)
(declare-fun tla__in (u u) Bool)

;; Terms, predicates and strings
(declare-fun a_CONSTANTunde_wunde_a () u)

(assert
  (forall ((X_3 Int)) (! (= (u2int (int2u X_3)) X_3) :pattern ((int2u X_3)))))
(assert
  (forall ((X_3 u))
  (! (= (tla__in X_3 tla__Int) (exists ((N_2 Int)) (= X_3 (int2u N_2)))) :pattern ((tla__in X_3 tla__Int)))))
(assert
  (forall ((M_1 Int) (N_2 Int))
  (! (= (tla__le (int2u M_1) (int2u N_2)) (<= M_1 N_2)) :pattern ((tla__le (int2u M_1) (int2u N_2))))))
(assert
  (forall ((S_6 u) (T_7 u))
  (=> (forall ((X_3 u)) (= (tla__in X_3 S_6) (tla__in X_3 T_7))) (= S_6 T_7))))

;; Theorem to be Proved
(assert (not true))
(assert (tla__in a_CONSTANTunde_wunde_a tla__Int))
(assert (tla__le (int2u 0) a_CONSTANTunde_wunde_a))

(check-sat)
(exit)

RESULT
unsat

(* created new ".tlacache/LevelComparisonCONSTANT_THEOREM_Scope_test.tlaps/LevelComparisonCONSTANT_THEOREM_Scope_test.thy" *)
(* fingerprints written in ".tlacache/LevelComparisonCONSTANT_THEOREM_Scope_test.tlaps/fingerprints" *)
